region/brinstar/kraid.json

```diff
@@ -653,7 +669,21 @@
           "to": [
             {
               "id": 2,
-              "requires": [ "canKillStandingGreenSpacePirates" ],
+              "requires": [
```

Comment:
Speed Booster also works. It's pretty annoying coming from the left, but someone good enough can probably avoid damage and jump over the pirates and kill mini kraid with Power Beam.

Reply:
That's a funny strat, waiting for the spike. Thanks to this I also filled in missing attacks for "Mini-Kraid" in the enemies json.

Reply:
There's a bit of short charging in that video though. I'm not sure it works otherwise.

Reply:
Got one after like 20 tries. It's really ridiculously tight. You have to go fast and land super close to the last one.

Reply:
It's only 7 damage of difference from using a spike anyway